specification/azurestack/resource-manager/Microsoft.AzureStack/preview/2020-06-01-preview/LinkedSubscription.json

```diff
@@ -0,0 +1,434 @@
+            "$ref": "#/parameters/LinkedSubscriptionNameParameter"
+          },
+          {
+            "$ref": "AzureStack.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LinkedSubscription"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "AzureStack.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete the requested Linked Subscription.": {
+            "$ref": "./examples/LinkedSubscription/Delete.json"
+          }
+        },
+        "tags": [
+          "LinkedSubscriptions"
+        ],
+        "description": "Delete the requested Linked Subscription resource.",
+        "operationId": "LinkedSubscriptions_Delete",
+        "parameters": [
+          {
+            "$ref": "AzureStack.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "AzureStack.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkedSubscriptionNameParameter"
+          },
+          {
+            "$ref": "AzureStack.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "204": {
+            "description": "NO CONTENT"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "AzureStack.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "x-ms-examples": {
+          "Create or update a Linked Subscription.": {
+            "$ref": "examples/LinkedSubscription/Put.json"
+          }
+        },
+        "tags": [
+          "LinkedSubscriptions"
+        ],
+        "description": "Create or update a linked subscription resource.",
+        "operationId": "LinkedSubscriptions_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "AzureStack.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "AzureStack.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/LinkedSubscriptionNameParameter"
+          },
+          {
+            "$ref": "AzureStack.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/NewLinkedSubscriptionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LinkedSubscription"
+            }
+          },
+          "201": {
```

Comment:
>"201" [](start = 10, length = 5)

Just to double check, this is meant to be a sync operation right?

Reply:
yes, it's a sync operation